src/main/java/competition/subsystems/armlifting/LiftingSubsystem.java

```diff
@@ -0,0 +1,53 @@
+package competition.subsystems.armlifting;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+ 
+@Singleton
+public class LiftingSubsystem extends BaseSubsystem {
+    
+    final DoubleProperty liftingPowerProp;
+    final DoubleProperty loweringPowerProp;
+    int maximumLiftHeight;
+    int currentLiftHeight;
```

Comment:
Seems like these should also be `DoubleProperty`

Reply:
I created a DoubleProperty for maximumLiftHeight and currentLiftHeight.